code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -64,6 +64,8 @@
 			if(toast)
 				M.add_atom_colour("#3A3A3A", FIXED_COLOUR_PRIORITY)
 				M.desc = "They're toast."
+				M.foodtypes &= ~ RAW //it's toast!
+				M.foodtypes |= FRIED
```

Comment:
foodtypes is used on examine and during Initialize when making it editable, adding this to the list after you made the food won't actually affect the food, just the description of it, you can look to https://github.com/tgstation/tgstation/pull/68319 to see how that worked out.

Reply:
i'm not sure that i follow